_rules/button-accessible-name-97a4e1.md

```diff
@@ -168,59 +168,32 @@ Off screen element without an [accessible name][].
 </html>
 ```
 
-#### Failed Example 4
-
-Button has an [accessible name][] that is only [whitespace][].
-
-```html
-<button></button>
-```
-
 ### Inapplicable
 
 #### Inapplicable Example 1
 
-Image buttons are tested in a different rule.
+Image buttons are tested in a separate rule, because also have to be tested for [success criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content).
 
 ```html
 <input type="image" value="download" alt="Download" />
 ```
 
 #### Inapplicable Example 2
 
-Not [visible](#visible) in page and not [included in the accessibility tree][].
+Buttons that are not [included in the accessibility tree][] are ignored by assistive technologies. These are not required to have an accessible name. If at some future state of the page the element is made visible, an [accessible name][] will be necessary.
 
 ```html
-<html>
-	<style>
-		.notInPage {
-			position: absolute;
-			left: -9999px;
-			top: -9999px;
-		}
-	</style>
-	<body>
-		<button class="notInPage" aria-hidden="true">Confirm</button>
-	</body>
-</html>
+<button style="display: none;"></button>
```

Comment:
```suggestion
<button aria-hidden="true"></button>
```
There's no need to make the button invisible for that rule to be inapplicable.

Reply:
There is. It would fail SC 4.1.2. Inapplicable examples have to satisfy the success criteria they test.